lib/src/DynamicProto.ts

```diff
@@ -425,6 +517,12 @@ export default function dynamicProto<DPType, DPCls>(theClass:DPCls, target:DPTyp
     // Note casting the same type as we don't actually have the base class here and this will provide some intellisense support
     delegateFunc(target, baseFuncs as DPType);
 
-    // Populate the Prototype for any overidden instance functions
-    _populatePrototype(classProto, className, target, instFuncs);
+    // Don't allow setting instance functions for older IE instances
+    let setInstanceFunc = !!_objGetPrototypeOf;
```

Comment:
IE8 and older doesn't have Object.getPrototypeOf() so this effectively disables the optimizations for IE, as we *may* not always be able to parse the prototype hierarchy completely.